### DISCUSSION TITLE
Is there a way to allow Parcel to support TS in node modules?

### DISCUSSION THREAD

**Discussion by jamesdavieswales** [parcel-bundler/parcel#7525] [Q&A]
In my workplace, we have the need to use some internal non transpiled TS that we share amongst projects through node modules. 

This ends up throwing errors as it's not reading the TSX / TS types correctly. Is there an option or something I can do like ts-loaders allowTsInNodeModules: true to allow this to work? Thanks!

**Comment by mischnic** [comment]
Looking at the implementation of Parcel 2, it's using Typescript based on the file extension and regardless of the actual file path.

Though I think this is more of an oversight and not really intentional, so we might change that in the future.
